bin/webpack.js

```diff
@@ -166,9 +166,11 @@ if(argv.init) {
 	if (!filePaths.length) {
 		throw new Error('Please specify a path to your webpack config');
 	}
+	const outputConfigName = filePaths[1] || `${filePaths[0]}v2.js`;
```

Comment:
Okay but in this case I'd suggest renaming the original config to something with `webpack.config.orig.js` in order to be able to run `webpack` without specifying the path to the new one in case of the default name. Either way, I _guess_ for most people the next operation after transforming would be to try run the build and see if everything works. With a different name it's not possible.